backend/flightData.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from datetime import datetime
+
+def get_flight_data(aircraft: str, distance: int):
+    url = f"https://despouy.ca/flight-fuel-api/q/?aircraft={aircraft}&distance={distance}"
+
+    params = {
+        "icao": aircraft,
+        "distance": distance
+    }
```

Comment:
Great! Are we sure that these are the only params?

Reply:
For now yes, we can also do batch queries, but I just want basic functionality for now